modules/nw-multus-ipam-object.adoc

```diff
@@ -310,4 +310,47 @@ pod/whereabouts-reconciler-p4sxw 1/1 Running 0 6s
 pod/whereabouts-reconciler-rvfdv 1/1 Running 0 6s
 pod/whereabouts-reconciler-svzw9 1/1 Running 0 6s
 daemonset.apps/whereabouts-reconciler 6 6 6 6 6 kubernetes.io/os=linux 6s
-----
\ No newline at end of file
+----
+
+[id="nw-multus-creating-whereabouts-dualstack-config_{context}"]
+== Creating a DualStack config
```

Comment:
nit: I think this can be "Dual stack" or "dual-stack" (unless we have a precedent set for DualStack in other docs)

Reply:
Also maybe we want to specifically say "Creating a configuration for assignment of dual-stack IP addresses dynamically" which is more specific to what Whereabouts actually does here. (maybe too wordy, but, as a start)

Reply:
Dualstack is frequently used as one word in docs but I will absolutely double-check to ensure this is the right context!